servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSHandler.java

```diff
@@ -96,4 +96,6 @@
     DomainRoleMembership getPendingDomainRoleMembersList(ResourceContext context, String principal);
     Schema getRdlSchema(ResourceContext context);
     ResourceContext newResourceContext(HttpServletRequest request, HttpServletResponse response);
+    void recordMetrics(String httpMethod, int httpStatus, String apiName, ResourceContext ctx);
```

Comment:
let's use the ResourceContext as the first argument to be consistent with other APIs